### ISSUE TITLE
Wrong focal point when zooming out

### ISSUE THREAD

**Issue by Guardiola31337** [mapbox/mapbox-gestures-android#40]
While testing [MapboxGLAndroidSDKTestApp](https://github.com/mapbox/mapbox-gl-native/tree/release-boba/platform/android/MapboxGLAndroidSDKTestApp) ([`release-boba` branch](https://github.com/mapbox/mapbox-gl-native/tree/release-boba)) found 👇

![wrong_focal_point_zooming_out](https://user-images.githubusercontent.com/1668582/38504581-556bdb78-3c15-11e8-87ee-28914a2a2625.gif)

This is the [`SimpleMapActivity`](https://github.com/mapbox/mapbox-gl-native/blob/release-boba/platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/SimpleMapActivity.java) running in a Nexus 5 (Android 6.0).

Steps to reproduce:

1. Launch `SimpleMapActivity`
2. Zoom in (quick double tap with 1 finger)
3. Zoom out (quick double tap with 2 fingers)
4. As shown above map scrolls when it shouldn't

Noting that it doesn’t reproduce always.

👀 @LukasPaczos

**Comment by LukasPaczos** [comment]
Thanks for capturing this @Guardiola31337!
> 4. As shown above map scrolls when it shouldn't

Noting, that it's not really the scroll issue, but the zoom out focal point is wrong.

**Comment by Guardiola31337** [comment]
> Noting, that it's not really the scroll issue, but the zoom out focal point is wrong.

Correct. This is the side effect tho 😉 (sorry if I didn't make myself clear).
